src/storekit.cs

```diff
@@ -245,6 +250,15 @@ interface SKPaymentQueue {
 		[NoWatch, NoTV, NoMac, iOS (13,4)]
 		[Export ("showPriceConsentIfNeeded")]
 		void ShowPriceConsentIfNeeded ();
+
+		[NoWatch, NoTV, NoMac, iOS (14,0)]
+		[Export ("presentCodeRedemptionSheet")]
+		void PresentCodeRedemptionSheet ();
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("transactionObservers")]
+		[Protocolize]
```

Comment:
we stopped using this after XAMCORE_2_0 - that was only useful when supporting *classic*
use the interface for the protocol instead

side note: we have warnings about exposing concrete (vs interface) types in bindings but I'm not 💯% sure they all work with `[Protocolize]`

Reply:
They do not, and I used because it was already in the file, doing it the right way.